docs/mine/lotus/split-markets-miners.md

```diff
@@ -126,3 +126,41 @@ export LOTUS_MINER_PATH=~/markets-repo-location
 ```sh
 LOTUS_MINER_PATH=~/markets-repo-location ./lotus-miner run
 ```
+
+## Rollback from a multi-process architecture back to `lotus-miner` monolith process
+
+In case you want to revert the changes listed above and go back to running `lotus-miner` as a single process, you have to do the following:
+
+1. Make sure that the `mining/sealing/proving` node is publicly exposed, as we will be enabling the markets subsystem on it.
```

Comment:
```suggestion
1. Make sure that the Lotus node is publicly exposed. We will be enabling the markets subsystem on it.
```

Reply:
During `rollback` (what this PR is adding docs for), the user has 2 miner nodes - the `mining/sealing/proving` miner node, and the `markets` miner node, and this chunk of docs is about uniting them into 1 miner node.

Further up in the docs I was referring to the `mining/sealing/proving` node with that name (sometimes in the past we've also used the `main` miner node, but that's not great...)

---

So I think this should stay as is, or at least we should be explicit about which repository config we are editing.